{{cookiecutter.github_repository}}/provisioner/roles/project_data/tasks/webpack.yml

```diff
@@ -0,0 +1,42 @@
+{% raw %}---
+- name: Download NodeSource Setup script
+  get_url:
+    url: "https://deb.nodesource.com/setup_{{ webpack_node_version }}.x"
+    dest: "/tmp/node_setup_{{ webpack_node_version }}.x"
+  become: false
+
+- name: Run NodeSource setup script
+  command: "bash /tmp/node_setup_{{ webpack_node_version }}.x"
+  become: true
+
+- name: Update apt-cache separately
+  shell: apt update
+  become: yes
+
+- name: Install Node.js
+  apt:
+    pkg: nodejs
+    state: present
+
+- name: Make sure webpack stats file exists
+  file:
+    path: "{{ project_path }}/webpack-stats.json"
+    state: touch
+    owner: "{{user}}"
+    group: "{{uwsgi_group}}"
+
+- name: Run build webpack
+  command: "{{ webpack_path }} --config {{ webpack_config }}"
+  become: false
+  tags: ['deploy']
+  args:
+   chdir: "{{ project_path }}"
+
+- name: collect static
+  django_manage:
+    command: "collectstatic --no-input -i js -i css -i server.js -i webpack.base.config.js -i webpack.dev.config.js -i webpack.prod.config.js"
```

Comment:
ignoring js and css makes  it too generic.
 collectstatic command starts ignoring all the js and css files even from django or any other apps.